airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/ConfigFetchActivityImpl.java

```diff
@@ -127,13 +131,36 @@ private ScheduleRetrieverOutput getTimeToWaitFromScheduleType(final StandardSync
         final Date nextRunStart = cronExpression.getNextValidTimeAfter(new Date(earliestNextRun));
         final Duration timeToWait = Duration.ofSeconds(
             Math.max(0, nextRunStart.getTime() / MS_PER_SECOND - currentSecondsSupplier.get()));
+
+        addSchedulingNoiseForAllowListedWorkspace(timeToWait, standardSync);
         return new ScheduleRetrieverOutput(timeToWait);
       } catch (final ParseException e) {
         throw (DateTimeException) new DateTimeException(e.getMessage()).initCause(e);
       }
     }
   }
 
+  private void addSchedulingNoiseForAllowListedWorkspace(Duration timeToWait, StandardSync standardSync) {
```

Comment:
is this activity used in AWS data plane? Seems it depends on ConfigRepository but data plane can't talk to db directly

Reply:
I believe that this activity runs in the control plane.

Reply:
(In any case the dependency already exists, so it's a bit unrelated to this PR.)